TgeeT/Questioner
# #163046735 meetup endpoint

#### What does this PR do?

Create an endpoint for meetups and allow a user to post a meet up and query the data stored
#### Description of Task to be completed?

write failing test, 
write the bear minimum code to pass the tests

#### How should this be manually tested?
How should this be manually tested?

follow the steps below:
1. git clone https://github.com/tgeet/Questioner.git
2. Create a virtual enviroment
    virtualenv venv
3. git checkout to develop branch
4. run pip3 install -r requirements.txt
5. export FLASK_APP=run.py
6. rflask run the app and use postman to verify

### What are the relevant pivotal tracker stories?
#163046735
#163046727
